daprdocs/content/en/operations/resiliency/resiliency-overview.md

```diff
@@ -164,5 +164,10 @@ spec:
 ```
 
 ## Related links
+
+<div class="embed-responsive embed-responsive-16by9">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/7D6HOU3Ms6g?start=184" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Use this heading and format for the embedding. Not the use of No-Cookies

Watch [this video for how to use resiliency policies](https://youtu.be/7D6HOU3Ms6g?t=184):

<iframe width="360" height="315" src="https://www.youtube-nocookie.com/embed/sembed/7D6HOU3Ms6g?start=184" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>